pyproject.toml

```diff
@@ -22,12 +23,13 @@ tomlkit = "^0.5.3"
 sphinx-autodoc-typehints = "^1.6"
 
 [tool.poetry.scripts]
-jira-make-linked-issue = "jiratools:_create_test_jira_from"
-jira-add-comment = "jiratools:_cli_add_comment"
-jira-example-config = "jiratools:_example_config_install"
-jira-search-issues = "jiratools:_cli_search"
-jira-link-issues = "jiratools:cli_jira_link"
-jira-update-assignee = "jiratools:_change_jira_assignee"
+jira-link-issues = "jiratools:jira_link"
+jira-make-linked-issue = "jiratools:make_linked"
+jira-add-comment = "jiratools:add_comment"
+jira-search-issues = "jiratools:search"
+jira-update-assignee = "jiratools:reassign"
```

Comment:
Comment for future PRs. I'm not sure what happened here, but the new list of "existing" commands is not in the same order as the olde list which made it really annoying to make sure nothing got dropped. Had the new list been in alpha order, that've been OK too, but it seems to be randomized?

Reply:
I think the list is in the order that commands were added to the package